### DISCUSSION TITLE
Change entries in config file to lower case

### DISCUSSION THREAD

**Discussion by Xol** [CerealDevelopment/CerealBot#32] [Ideas]
Hey guys,

this is more of a proposal than an idea. I'd like to change the content in the config file to lower case (or camel case to be exact). This is mainly an aesthetic change, but it would be also more like the regular import statements.

```javascript
import { resourceEndsWith, getCerealColor, getRandomNumber } from "../../utils";
import { IMGUR } from "../../../config.json";
```

**Comment by jbgerth** [comment]
I do like having the all caps for configs. It makes it easy, to find the config in the code later on. That being said I don't mind changing it 🤷‍♂️

**Reply by Xol** [reply]
Hm... that's true. I didn't think it fully through. You're right. In the end, they are just like normal constant variables and they are also written in upper case. And in upper case they are fast to find and to identified as constant (config) variable.

Let's forget it. 😄
